tekton_kfp_guide.md

```diff
@@ -1,88 +1,319 @@
-    ```shell
-    cd $GOPATH/src/go/github.com/kubeflow
-    git clone github.com/kubeflow/kfp-tekton
-    mv kfp-tekton pipelines
-    cd pipelines
-    ```
-
-2. For local binary builds, use the `go build` commands
-   ```shell
-   go build -o apiserver ./backend/src/apiserver
-   go build -o agent ./backend/src/agent/persistence
-   go build -o workflow ./backend/src/crd/controller/scheduledworkflow/*.go
-   ```
-
-   Note: The metadata writer is written in Python, so the code will be compiled during runtime execution.
-
-3. For Docker builds, use the below `docker build` commands
-   ```shell
-   DOCKER_REGISTRY="<fill in your docker registry here>"
-   docker build -t ${DOCKER_REGISTRY}/api-server -f backend/Dockerfile .
-   docker build -t ${DOCKER_REGISTRY}/persistenceagent -f backend/Dockerfile.persistenceagent .
-   docker build -t ${DOCKER_REGISTRY}/metadata-writer -f backend/metadata_writer/Dockerfile .
-   docker build -t ${DOCKER_REGISTRY}/scheduledworkflow -f backend/Dockerfile.scheduledworkflow .
-   ```
-
-4. Push the images to registry and modify the Kustomization to use your own built images.
+# Set the path to the base directory where you want to store one or more 
+# Kubeflow deployments. For example, /opt/.
+# Then set the Kubeflow application directory for this deployment.
+export BASE_DIR=<path to a base directory>
+export KF_DIR=${BASE_DIR}/${KF_NAME}
+
+# Set the configuration file to use, such as the file specified below:
+export CONFIG_URI="https://raw.githubusercontent.com/IBM/KubeflowDojo/master/manifests/kfctl_ibm_tekton.v1.1.0.yaml"
+
+# Generate and deploy Kubeflow:
+mkdir -p ${KF_DIR}
+cd ${KF_DIR}
+kfctl apply -V -f ${CONFIG_URI}
+```
+
+* **${KF_NAME}** - The name of your Kubeflow deployment.
+  If you want a custom deployment name, specify that name here.
+  For example,  `my-kubeflow` or `kf-test`.
+  The value of KF_NAME must consist of lower case alphanumeric characters or
+  '-', and must start and end with an alphanumeric character.
+  The value of this variable cannot be greater than 25 characters. It must
+  contain just a name, not a directory path.
+  This value also becomes the name of the directory where your Kubeflow 
+  configurations are stored, that is, the Kubeflow application directory. 
+
+* **${KF_DIR}** - The full path to your Kubeflow application directory.
+
+### Multi-user, auth-enabled
+Run the following commands to deploy Kubeflow with GitHub OAuth application as authentication provider by dex. To support multi-users with authentication enabled, this guide uses [dex](https://github.com/dexidp/dex) with [GitHub OAuth](https://developer.github.com/apps/building-oauth-apps/). Before continue, refer to the guide [Creating an OAuth App](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/) for steps to create an OAuth app on GitHub.com.
```

Comment:
Is there a reason you are using the old Github developer docs or can we use the new ones?
```suggestion
Run the following commands to deploy Kubeflow with GitHub OAuth application as authentication provider by dex. To support multi-users with authentication enabled, this guide uses [dex](https://github.com/dexidp/dex) with [GitHub OAuth](https://docs.github.com/developers/apps/building-oauth-apps). Before continue, refer to the guide [Creating an OAuth App](https://docs.github.com/developers/apps/creating-an-oauth-app) for steps to create an OAuth app on GitHub.com.
```